test/jb_projects/claim_handle.test.js

```diff
@@ -0,0 +1,297 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+import { fastForward, getTimestamp } from '../helpers/utils';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import jbOperatoreStore from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+
+describe('JBProjects::claimHandle(...)', function () {
+  const PROJECT_HANDLE_1 = 'PROJECT_1';
+  const PROJECT_HANDLE_2 = 'PROJECT_2';
+  const PROJECT_HANDLE_3 = 'PROJECT_3';
+  const METADATA_CID = '';
+  const PROJECT_ID = 1;
+  const PROJECT_ID_2 = 2;
+
+  let jbOperatorStore;
+  let CLAIM_HANDLE_PERMISSION_INDEX;
+
+  beforeEach(async function () {
+    let jbOperatorStoreFactory = await ethers.getContractFactory('JBOperatorStore');
+    jbOperatorStore = await jbOperatorStoreFactory.deploy();
+
```

Comment:
operator store should be removed from beforeEach since it is stateful. the operations/permission index can just be wrapped in before(async... because it won't change.